CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/ImageGraphBuilderPlugin.java

```diff
@@ -247,6 +248,8 @@ protected void edit(final GraphWriteMethods graph, final PluginInteraction inter
                     graph.setObjectValue(vertexColorAttributeId, vxId, color);
 
                     if (addLayers) {
+                        // get layer bitmask attribute          
+                        final int vertexBitmaskAttributeId = LayersConcept.VertexAttribute.LAYER_MASK.ensure(graph);
```

Comment:
This should be defined before entering the nested for loops.
Should not be obtaining/ensuring the attributeId separately for each and every pixel in the image.